setup1-shared/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2018"
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+tokio = { version = "1.7", features = ["io-util"] }
```

Comment:
is it ok for all dependencies using setup1-shared to pull in tokio? I can't remember if the wasm contributor uses it

Reply:
Nice catch! We don't use `setup1-shared` in a browser contributor yet, but most likely will use it. Let me see if I can make it optional with a feature